.github/workflows/coverity.yml

```diff
@@ -8,15 +8,17 @@ on:
   workflow_dispatch:
 jobs:
   coverity:
-    if: github.repository == 'valkey-io/valkey'
+    if: github.repository == 'madolson/valkey'
```

Comment:
Should this change be added to the upstream?

Reply:
Let me change it back to a draft, I was testing stuff on the same branch as this pr since it wasn't working correctly.